### ISSUE TITLE
Implement `check` command to execute controls 

### ISSUE THREAD

**Issue by kaidaguerre** [turbot/steampipe#407]
One or more controls may be run with the steampipe check command. Note that steampipe query, steampipe check, steampipe report, and steampipe run are separate sub-commands so they have their own arguments and flags:
```
steampipe check aws    # all aws controls (all checks in the mod)
steampipe check 'aws.control.cisv130*'     # all aws CIS v 1.3.0 controls 
steampipe check 'aws.control.cisv130_1_*'  # all section 1 
steampipe check aws.control.cisv130_1_1    # only control 1_1
```

**Comment by kaidaguerre** [comment]
Will be very similar to query command, and should try to share code as far as possible

```
identify controls
for each control query
   - execute
   - stream results
   ```

**Comment by kaidaguerre** [comment]
closed by 885db7fc7d92de1be5e5b31fe3b0df1303746faf
